sky/skylet/job_lib.py

```diff
@@ -55,7 +55,7 @@ def create_table(cursor, conn):
         submitted_at FLOAT,
         status TEXT,
         run_timestamp TEXT CANDIDATE KEY,
-        start_at FLOAT)""")
+        start_at FLOAT DEFAULT -1)""")
```

Comment:
Can we update L73 and related comments?  E.g., at least the transition `INIT -> FAILED` should be captured.

Reply:
Updated. Thanks!